<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tvWindow</class>
 <widget class="QMainWindow" name="tvWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1951</width>
    <height>1210</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="windowTitle">
   <string>TV Control</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>1351</width>
      <height>241</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:48pt;&quot;&gt;Choose Stimulus type&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>290</y>
      <width>701</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:26pt;&quot;&gt;On-Off Setting&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>1440</x>
      <y>10</y>
      <width>581</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;08SeoulHangang&quot;;
</string>
    </property>
    <property name="text">
     <string>On-Off : one by one</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1510</x>
      <y>540</y>
      <width>281</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1510</x>
      <y>780</y>
      <width>281</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>420</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dark time</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>420</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Flash Time</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_3">
    <property name="geometry">
     <rect>
      <x>1440</x>
      <y>220</y>
      <width>581</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;08SeoulHangang&quot;;</string>
    </property>
    <property name="text">
     <string>On-Off : row column</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>660</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>training</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>650</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>testing</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_6">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>690</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;08SeoulHangang&quot;;</string>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_7">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>690</y>
      <width>281</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;08SeoulHangang&quot;;</string>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>80</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>430</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>repeats</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_8">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>470</y>
      <width>251</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;08SeoulHangang&quot;;</string>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_9">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>680</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;08SeoulHangang&quot;;</string>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_10">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>770</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;08SeoulHangang&quot;;</string>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>7</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_11">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>860</y>
      <width>201</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;08SeoulHangang&quot;;</string>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>3</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>480</y>
      <width>231</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;08SeoulHangang&quot;;</string>
    </property>
    <property name="maximum">
     <double>5000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>10.000000000000000</double>
    </property>
    <property name="value">
     <double>150.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinBox_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>470</y>
      <width>271</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;08SeoulHangang&quot;;</string>
    </property>
    <property name="maximum">
     <double>5000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>10.000000000000000</double>
    </property>
    <property name="value">
     <double>150.000000000000000</double>
    </property>
   </widget>
   <zorder>label_4</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label</zorder>
   <zorder>radioButton</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>radioButton_3</zorder>
   <zorder>label_11</zorder>
   <zorder>label_12</zorder>
   <zorder>spinBox_6</zorder>
   <zorder>spinBox_7</zorder>
   <zorder>label_13</zorder>
   <zorder>spinBox_8</zorder>
   <zorder>spinBox_9</zorder>
   <zorder>spinBox_10</zorder>
   <zorder>spinBox_11</zorder>
   <zorder>spinBox</zorder>
   <zorder>spinBox_2</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
